test/services/katello/pulp/repository/file_test.rb

```diff
@@ -10,10 +10,30 @@ class FileBaseTest < ::ActiveSupport::TestCase
         def setup
           @master = FactoryBot.create(:smart_proxy, :default_smart_proxy)
           @custom = katello_repositories(:generic_file)
+          @repo = katello_repositories(:pulp3_file_1)
+          RepositorySupport.create_and_sync_repo(@repo)
+        end
```

Comment:
We had a fixture for files but we weren't saving it to DB in earlier tests. Hence we missed the error in distribution path and the label containing a blank space. With this test we are saving,syncing and indexing in VCR test which highlighted these issues in our fixtures.